tests/integration_tests/test_gym_envs.py

```diff
@@ -0,0 +1,130 @@
+from itertools import product
+from multiprocessing import Process
+import pickle
+import unittest
+
+import gym
+from nose2.tools import params
+
+from garage.tf.envs.base import TfEnv
+from garage.theano.envs.base import TheanoEnv
+
+# White list of environments that cannot be rendered
+not_rendering_envs = [
+    "Blackjack-v0", "NChain-v0", "Roulette-v0", "GuessingGame-v0",
+    "HotterColder-v0", "Blackjack-v0", "NChain-v0", "Roulette-v0",
+    "GuessingGame-v0", "HotterColder-v0"
+]
+
+# White list of environments that cannot be closed
+not_closing_envs = [
```

Comment:
Please use the pattern I implemented (but didn't commit) here https://github.com/rlworkgroup/garage/blob/env_pickle_tests/tests/quirks.py
https://github.com/rlworkgroup/garage/blob/env_pickle_tests/tests/helpers.py

In fact, I think you will find that entire `test/` tree useful.